pkg/ebpf/c/tracee.bpf.c

```diff
@@ -190,7 +191,8 @@ Copyright (C) Aqua Security inc.
 #define SECURITY_INODE_SYMLINK          1031
 #define SOCKET_DUP                      1032
 #define HIDDEN_INODES                   1033
-#define MAX_EVENT_ID                    1034
+#define DETECT_HOOKED_SYSCALLS          1034
```

Comment:
The event (at least in the BPF part) isn't related to hooks, so you should change the name to suit its purpose better.
Maybe `SYSCALLS_INFO`? Think on a name that you think match the purpose here.